environments/manager/secrets.yml

```diff
@@ -1,2 +1,4 @@
 ---
 netbox_api_token: "0000000000000000000000000000000000000000"
+
+manager_listener_broker_password: BO6yGAAq9eqA7IKqeBdtAEO7aJuNu4zfbhtnRo8Y
```

Comment:
Can we use "password" here like in other locations to make it obvious that this is not secure and only for testing?

Reply:
This is the password from the Rabbitmq account in OpenStack. It's fine for me to changes that. But I would prefer to do this in a separate PR.